packages/frontend/components/composed/issue/IssueSummary/IssueSummary.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useMemo } from "react";
+import { graphql } from "react-relay";
+import { useMaybeFragment } from "@wdp/lib/api/hooks";
+import { useRouter } from "next/router";
+import { routeQueryArrayToString } from "@wdp/lib/routes";
+import IssueSummaryOrderingList from "./IssueSummaryOrderingList";
+import IssueSummaryList from "./IssueSummaryList";
+import { IssueSummaryFragment$key } from "@/relay/IssueSummaryFragment.graphql";
+
+export default function IssueSummary({ data }: Props) {
+  const content = useMaybeFragment(fragment, data);
+  const router = useRouter();
+  const order = useMemo(() => {
+    return routeQueryArrayToString(router.query.order);
+  }, [router.query]);
+
+  return order ? (
+    <IssueSummaryOrderingList data={content?.ordering?.children} />
+  ) : (
+    <IssueSummaryList data={content?.items} />
+  );
+}
+
+interface Props {
+  data?: IssueSummaryFragment$key | null;
+}
+
+const fragment = graphql`
+  fragment IssueSummaryFragment on Collection {
+    items {
+      ...IssueSummaryListFragment
+    }
+    ordering(identifier: $orderIdentifier) {
```

Comment:
I think it might be safe to assume that we want to always use ordering on this page. In that case, the frontend or the API will need to know what the default ordering is.